apache-php-mysql/README.md

```diff
@@ -73,6 +68,10 @@ On Webssh run the command below to check if the startup logs from entrypoint.sh
 - **Version 0.4**
   1. Update the section Startup Log in README.md.
   2. Create default database - azurelocaldb.(You need set DATABASE_TYPE to **"local"**)
+  3. Considering security, please set database authentication info on [*"App settings"*](#deploying-on-azure) when enable **"local"** mode.   
+     Note: Database authentication is also used by phpMyAdmin.
```

Comment:
Note: the credentials below is also used by phpMyAdmin

Reply:
got it